src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceLookup/ILEmit/ILEmitResolverBuilderContext.cs

```diff
@@ -9,8 +9,9 @@ namespace Microsoft.Extensions.DependencyInjection.ServiceLookup
 {
     internal sealed class ILEmitResolverBuilderContext
     {
+        public ILEmitResolverBuilderContext(ILGenerator generator) => Generator = generator;
         public ILGenerator Generator { get; set; }
```

Comment:
```suggestion
        public ILGenerator Generator { get; }
```

We don't need this setter anymore, right?